### ISSUE TITLE
Dot elements change xml:id

### ISSUE THREAD

**Issue by martha-thomae** [MeasuringPolyphony/mp_editor#70]
When uploading a parts-MEI file, the `@xml:id` values of the `<dot>` elements are not preserved, they change when exporting either the parts or the score MEI files. Here is a file to test this.

[33v-34r.mei.zip](https://github.com/MeasuringPolyphony/mp_editor/files/5249038/33v-34r.mei.zip)

**Comment by martha-thomae** [comment]
Same issue with `<accid>` elements

**Comment by karend27** [comment]
Hi @martha-thomae - is this still an issue?

**Comment by martha-thomae** [comment]
Yes, this is still an issue. It also happens with the ids of `<ligature>`, `<surface>`, `<graphic>`, and `<zone>`.

**Comment by martha-thomae** [comment]
Another proposed solution is to go directly into the score editor when uploading a file coming from another application. (@JRegimbal  will help me look into this :) )
